common/script/ops/clearPMs.js

```diff
@@ -1,6 +1,6 @@
 module.exports = function clearPMs (user) {
   user.inbox.messages = {};
-  user.markModified('inbox.messages');
+  // user.markModified('inbox.messages');
```

Comment:
it's possible that the function is called before the user is wrapped.. but anyway we can't really remove the markModified calls